src/libs/ReportActionsUtils.js

```diff
@@ -38,37 +62,20 @@ function isDeletedAction(reportAction) {
     return message.length === 0 || lodashGet(message, [0, 'html']) === '';
 }
 
-/**
- * Sorts the report actions by sequence number, filters out any that should not be shown and formats them for display.
- *
- * @param {Array} reportActions
- * @returns {Array}
- */
-function getSortedReportActions(reportActions) {
-    return _.chain(reportActions)
-        .sortBy('sequenceNumber')
-        .filter(action => action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU
-
-            // All comment actions are shown unless they are deleted and non-pending
-            || (action.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && (!isDeletedAction(action) || !_.isEmpty(action.pendingAction)))
-            || action.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED
-            || action.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED)
-        .map((item, index) => ({action: item, index}))
-        .value()
-        .reverse();
-}
-
 /**
  * Finds most recent IOU report action number.
  *
  * @param {Array} reportActions
  * @returns {String}
  */
 function getMostRecentIOUReportActionID(reportActions) {
-    return _.chain(reportActions)
-        .where({actionName: CONST.REPORT.ACTIONS.TYPE.IOU})
-        .max(action => action.sequenceNumber)
-        .value().reportActionID;
+    const iouActions = _.where(reportActions, {actionName: CONST.REPORT.ACTIONS.TYPE.IOU});
+    if (_.isEmpty(iouActions)) {
+        return null;
+    }
+
+    sortReportActions(iouActions);
+    return _.last(iouActions).reportActionID;
```

Comment:
```suggestion
    return _.last(sortReportActions(iouActions)).reportActionID;
```
Use the return value of `sortReportActions(iouActions);`

Reply:
Good call, done